lyra/bot.json

```diff
@@ -210,36 +210,34 @@
-    "Clear": "Remove the queue and stop the song",
-    "Clean": "Clean bot message and any message related to the bot commands",
-    "Ping": "Get the bot latency",
-    "Loop": "Repeat the queue or the song",
-    "Shuffle": "Shuffles the queue",
-    "Move": "Move a song to the top of the queue or to specific position",
-    "Removeduplicate": "Remove duplicate songs from queue",
-    "Leave": "Leave the channel",
-    "Search": "Search in Youtube",
-    "Seek": "Change the position of the track or to specific time",
-    "Prefix": "Displays the bot server prefix or change it",
-    "Perms": "Displays the permissions of a user or role",
-    "allow": "Allow a user or role from using the bot",
-    "Deny": "Deny a user or role from using the bot",
-    "topmusic": "Show Must hear music in your bot or server or Lyra Cloud",
-    "texttoggle": "Denied the bot from responding in specific channels",
+    "Skip": "Skip the current playing track or any track in queue.",
+    "Back": "Plays previous track.",
+    "Volume": "Change the volume percent",
+    "Nowplaying": "Shows details of current playing track.",
+    "Clear": "Removes all tracks from the queue and stop the player.",
+    "Clean": "Clean up bot message and any messages linked to bot commands.",
+    "Ping": "Shows bot's Latency.",
+    "Loop": "Toogle looping your currently playing track.",
+    "Shuffle": "Shuffle the queue.",
+    "Move": "Move a song to the top of the queue or to specific position.",
+    "Removeduplicate": "Removes duplicated songs from queue.",
+    "Leave": "Leave the voice channel and reset the player.",
+    "Search": "Search for the input and returns a list of results for you to pick from.",
+    "Seek": "Skips to the specified timestamp in the currently playing track.",
+    "Prefix": "Show or edit the bot's server prefix.",
+    "Perms": "Displays user/role permissions.",
+    "topmusic": "Show most played music in your bot or server or Lyra Cloud",
+    "texttoggle": "Prevent bots from responding to specific channels.",
     "setlang": "Change the bot language",
-    "addadmin": "Add admin to your bot 'You can use the main bot to add the admin to all bots'",
-    "deladmin": "Remove admin from your bot 'You can use the main bot to remove the admin from all bots'",
-    "addserver": "Allow the bot enter the server 'You can use the main bot to allow the server to all bots'",
-    "connect": "Make all of your bots data like this bot except the prefix",
-    "vip": "Show subscription end date"
+    "addadmin": "Add admin to your bot 'You can use the main bot instead to add the admin to all bots'",
+    "deladmin": "Remove admin from your bot 'You can use the main bot instead to remove the admin from all bots'",
+    "addserver": "Give bot access to join multiple servers 'You can use the main bot instead to give access to all bots.'",
+    "connect": "Make all your bot data just like this bot, except the prefix.",
+    "vip": "View the subscription end date."
```

Comment:
```suggestion
    "vip": "Displays info about your subscription."
```

Reply:
Try this one, seems better for me. 
Shows details of your subscription.

Reply:
even better, **Shows details of your current subscription.**